plugins/00_dokku-standard/subcommands/run

```diff
@@ -23,6 +24,16 @@ dokku_run_cmd() {
   [[ "$DOKKU_RM_CONTAINER" ]] && local DOKKU_RUN_OPTS="--rm"
   has_tty && local DOKKU_RUN_OPTS+=" -i -t"
   is_image_herokuish_based "$IMAGE" && local EXEC_CMD="/exec"
+
+  # TODO: should we cleanup the procfile after run?
+  extract_procfile "$APP"
```

Comment:
You could set `$DOKKU_QUIET_OUTPUT` before the call and unset it afterwards. **shrug**